src/main/java/de/sandstorm_projects/telegramAlert/config/TelegramAlarmCallbackConfig.java

```diff
@@ -44,6 +44,16 @@ public static ConfigurationRequest createRequest() {
                 "See https://core.telegram.org/bots/api#formatting-options for more information on formatting.",
                 ConfigurationField.Optional.NOT_OPTIONAL
         ));
+        Map<String, String> postMode = new HashMap<>(2);
+        postMode.put("url", "URL query string");
+        postMode.put("json", "application/json");
+        configurationRequest.addField(new DropdownField(
+                Config.POST_MODE, "Post Mode", "url", postMode,
```

Comment:
I'm not sure if we need a config option for this, most users won't care and don't know what to pick.

I think we don't need an option for this, always using the "application/json"-method should be fine, I don't know of any reasons why someone would need the "URL query string"-method.

Reply:
Ok, I'm replacing old way to post messages to the new